src/Tizen.NUI/src/public/GraphicsTypeConverter.cs

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright(c) 2019 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Globalization;
+
+namespace Tizen.NUI
+{
+
+    /// <summary>
+    /// GraphicsTypeConverter class to convert types.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
```

Comment:
Please change it everywhere to <since_tizen> 6 </since_tizen>, because current API version is 6.